extension/chrome/elements/pgp_block_modules/pgp-block-render-module.ts

```diff
@@ -14,15 +14,128 @@ import { Ui } from '../../../js/common/browser/ui.js';
 import { Xss } from '../../../js/common/platform/xss.js';
 import { MsgBlockParser } from '../../../js/common/core/msg-block-parser.js';
 import { AcctStore } from '../../../js/common/platform/store/acct-store.js';
+import { GmailParser } from '../../../js/common/api/email-provider/gmail/gmail-parser.js';
+import { Str } from '../../../js/common/core/common.js';
 
 export class PgpBlockViewRenderModule {
   public doNotSetStateAsReadyYet = false;
 
   private heightHist: number[] = [];
+  private printMailInfoHtml!: string;
 
   constructor(private view: PgpBlockView) {
   }
 
+  public initPrintView = async () => {
+    const fullName = await AcctStore.get(this.view.acctEmail, ['full_name']);
+    Xss.sanitizeRender('.print_user_email', `<b>${fullName.full_name}</b> &lt;${this.view.acctEmail}&gt;`);
+    try {
+      const gmailMsg = await this.view.gmail.msgGet(this.view.msgId!, 'full', undefined);
+      const sentDate = new Date(GmailParser.findHeader(gmailMsg, 'date') ?? '');
+      const sentDateStr = Str.fromDate(sentDate).replace(' ', ' at ');
+      const from = Str.parseEmail(GmailParser.findHeader(gmailMsg, 'from') ?? '');
+      const fromHtml = from.name ? `<b>${from.name}</b> &lt;${from.email}&gt;` : from.email;
+      const ccString = GmailParser.findHeader(gmailMsg, 'cc') ? `Cc: <span data-test="print-cc">${Xss.escape(GmailParser.findHeader(gmailMsg, 'cc')!)}</span><br/>` : '';
+      const bccString = GmailParser.findHeader(gmailMsg, 'bcc') ? `Bcc: <span>${Xss.escape(GmailParser.findHeader(gmailMsg, 'bcc')!)}</span><br/>` : '';
+      this.printMailInfoHtml = `
+      <hr>
+      <p class="subject-label" data-test="print-subject">${GmailParser.findHeader(gmailMsg, 'subject')}</p>
```

Comment:
I think we should sanitize the `subject` and the `from.name` as Google previously allowed special chars within the name field such as this one: https://www.youtube.com/user/securityteamvictim

Someone could change the UI by injecting HTML within the subject (limited tags + attributes):
![image](https://user-images.githubusercontent.com/46025304/184573574-eb08da60-62e5-4d3f-b2d1-a37291f74935.png)

Reply:
Thank you for your notice. Just sanitized the above elements

Reply:
Thank you, too. sir